packages/node/__tests__/integration.spec.ts

```diff
@@ -32,6 +32,7 @@ describe('Disable Introspection with plugin', () => {
     })
 
     expect(response.statusCode).toBe(400)
+    expect(response.headers['content-type']).toBe('application/json')
     const body = JSON.parse(response.text)
     expect(body.data).toBeUndefined()
```

Comment:
In our server without yoga then we expect body.data to be null here. 
```suggestion
    expect(body.data).toBeNull()
```

What is technically/semantically more correct? presence of `data` and it is null, or lack of `data` as you have today.